### PR TITLE
Update Docs For v0.19.8 Release

### PR DESCRIPTION
**Pull request by seanmalloy**

### CODE REVIEW

**Comment by seanmalloy** [comment]
Table formatting is messed up.

/hold

**Comment by seanmalloy** [comment]
/kind documentation
/assign @Huang-Wei 
/cc @denkensk 
/hold cancel

I'll cherry-pick these doc changes to the `release-1.19` branch after this is merged to master.

File: README.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ Repository for out-of-tree scheduler plugins based on scheduler framework.
 
 Starting with release v0.18.9 container images are available in the official k8s container registry
```
Comment (Huang-Wei) [inline README.md:7]: Although v0.19.8 is not officially available yet, should we temporarily replace v0.18.9 with v0.19.8 here?

File: README.md [hunk 2]
```diff
@@ -6,14 +6,14 @@ Repository for out-of-tree scheduler plugins based on scheduler framework.
...
 `k8s.gcr.io/scheduler-plugins/kube-scheduler`. See the [Compatibility Matrix section](#compatibility-matrix)
-for the list of images.
+for the list of images. Also, starting with scheduler-plugins release v0.19.8 multi-arch container images are
```
Comment (denkensk) [inline README.md:9]: A little advice is not necessary: can we replace this paragraph with the following chart? Just add a column
like this:
Scheduler Plugins  | Compiled With k8s Version | Container Image                                     |Arch         |
-------------------|---------------------------|-----------------------------------------------------|-----|
v0.19.8            | v1.19.8                   | k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.19.8 |AMD64 ARM64 |
v0.18.9            | v1.18.9                   | k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.18.9 |AMD64 |

**Review by denkensk — COMMENTED** [review]
Thanks for your work

File: README.md (hunk 2 above)
Comment (seanmalloy) [reply README.md]: Fixed in commit fa0ae5f3e69206e49ecaae934affb304cef1a48f

File: README.md (hunk 1 above)
Comment (seanmalloy) [reply README.md]: I went with the suggestion from @denkensk to just put all of the versions and architectures into the table down below. This way we do not need to updated this paragraph for each release.

**Comment by seanmalloy** [comment]
@Huang-Wei and @denkensk this PR is ready for review again. Let me know if any changes are needed. Thanks!

**Comment by denkensk** [comment]
Thanks 
/lgtm

**Comment by seanmalloy** [comment]
I can squash commits after final review and before merge, so commit history is a bit cleaner.

**Comment by Huang-Wei** [comment]
> I can squash commits after final review and before merge, so commit history is a bit cleaner.

LGTM. Please squash.

**Comment by seanmalloy** [comment]
> > I can squash commits after final review and before merge, so commit history is a bit cleaner.
> 
> LGTM. Please squash.

@Huang-Wei and @denkensk squashed to a single commit. Ready for approval.

**Pull request merged**